BP/common/culture/cultures/00_magyar.txt

```diff
@@ -0,0 +1,428 @@
+﻿magyar_group = {
```

Comment:
Has anything been modified in this file from vanilla? If no we don't need it in the mod?

Reply:
No, nothing has